packages/state/recoil/selectors/misc.ts

```diff
@@ -0,0 +1,23 @@
+import { selectorFamily } from 'recoil'
+
+export const averageColorSelector = selectorFamily({
+  key: 'averageColor',
+  get: (url: string) => async () => {
+    if (!url || url.startsWith('http://localhost')) {
+      return undefined
+    }
+
+    try {
+      const response = await fetch(`https://fac.withoutdoing.com/${url}`)
```

Comment:
This hardcoded URL is sus. Can we move this to a config somewhere? That would let us document better what these endpoints are and do. (Will be helpful when we inevitably do more orchestration around the APIs we depend on).